prompting-client/src/lib.rs

```diff
@@ -11,6 +22,12 @@ mod socket_client;
 mod util;
 
 pub(crate) const SNAP_NAME: &str = "prompting-client";
+pub const SOCKET_ENV_VAR: &str = "PROMPTING_CLIENT_SOCKET";
+pub const DEFAULT_LOG_LEVEL: &str = "info";
+
+pub fn log_filter(filter: &str) -> String {
+    format!("{filter},hyper=error,h2=error")
+}
```

Comment:
See here.

I would prefer if there were an enum of support log levels, and if the clap arg were defined so the default is explicit.